django-backend/fecfiler/f3x_summaries/views.py

```diff
@@ -48,7 +48,7 @@ def get_queryset(self):
         report_id = (
             (
                 self.request.query_params.get("report_id")
-                or self.request.data["report_id"]
+                or ("report_id" in self.request.data and self.request.data["report_id"])
```

Comment:
you could use the same dictionary get `self.request.data.get("report_id"])` as the line above. `("report_id" in self.request.data and self.request.data["report_id"])` will return `report_id` if it's present, but `False` if it's not, which isn't desired